### ISSUE TITLE
RealNVP banana exersice failure :(

### ISSUE THREAD

**Issue by jecampagne** [kaijennissen/Normalizing_Flows#1]
Hello,
Iam trying to reproduce the RealNVP exercice with the banana toy model. I have tried different configs but all get a diverging validation loss, even with lowering the Adam learning rate. I have put my nb on Colab may be you can tell me what is going wrong.

Conercing the library version here is what I get
```python
print(tfp.__version__, distrax.__version__, optax.__version__, hk.__version__)
```
```
0.16.0 0.1.2 0.1.3 0.0.8
```

[see Google Colab nb](https://colab.research.google.com/drive/1FG1eRnZj8ie9BcwqRy6tugeBDeW3tbk-?usp=sharing)

Hope that you can help me as I would like to show friends how `distrax` is working on NFlows. Thanks

**Comment by kaijennissen** [comment]
@jecampagne Sorry for the delay. I cannot access the notebook. I guess you need to publish it first.
